src/install/postgresql/default-requirements.mdx

```diff
@@ -0,0 +1,12 @@
+---
+headingText: Check the compatibility and requirements
+componentType: default
+---
+
+
+* Our integration is compatible with PostgreSQL until version 16.
```

Comment:
```suggestion
* Our integration is compatible with PostgreSQL version 15.6 and lower. 
```

Reply:
I always forget about this style guide rule! https://docs.newrelic.com/docs/style-guide/word-choice/usage-dictionary/#version

Reply:
Good catch!